### PR TITLE
db creation

### PR DESCRIPTION
**Pull request by clemcx**
database creation

### CODE REVIEW

**Review by RMayeux — APPROVED** [review]
Tout est bon pour moi

File: src/Entity/Advisor.php [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+     */
+    private $picture;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $presentation;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\Tag", mappedBy="advisor")
```
Comment (benj-san) [inline src/Entity/Advisor.php]: mapped by advisors since it's a many to many relationship

File: src/Entity/Advisor.php [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+     */
+    private $tags;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\Board", inversedBy="advisors")
+     */
+    private $board;
```
Comment (benj-san) [inline src/Entity/Advisor.php]: boards since it's a many to many relationship

File: src/Entity/Tag.php [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $category;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\Advisor", inversedBy="tags")
+     */
+    private $advisor;
```
Comment (benj-san) [inline src/Entity/Tag.php]: advisors

**Review by benj-san — COMMENTED** [review]
Please fix the issues before I merge

**Review by benj-san — APPROVED** [review]

**Pull request merged**